gustaf/spline/visualize.py

```diff
@@ -247,10 +267,14 @@ def _vedo_showable(spline):
         if spline.para_dim != 1:
             c_mesh = c_mesh.to_edges(unique=True)
 
-        c_mesh.show_options["c"] = "red"
-        c_mesh.show_options["lw"] = 4
+        c_mesh.show_options["c"] = spline.show_options.get(
+            "control_mesh_c", "red"
+        )
+        c_mesh.show_options["lw"] = spline.show_options.get(
+            "control_mesh_lw", 4
+        )
         c_mesh.show_options["alpha"] = spline.show_options.get(
-            "control_points_alpha", 0.8
+            "control_points_alpha", 1.0
```

Comment:
how important is this?

Reply:
not, just thought it was a bug

Reply:
so, like 3?

Reply:
I think original intention was to make sure that spline body is prioritized

Reply:
```suggestion
            "control_points_alpha", 0.8
```